docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,42 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work.
+.. note::
+
+    If you ever encounter a link that helped you or that is not
```

Comment:
I still don't think this portion warrants a box.

The Notes box is visually intrusive, and should be reserved for something of utmost importance.

Suggesting people to submit a new link is just a suggestion, and not absolutely necessary.
If they miss reading this particular paragraph the first time, it's fine for me. 
The most important part of this document is the list of links below.